bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/i18n/LocaleProvider.java

```diff
@@ -25,4 +28,10 @@
      */
     Locale getLocale();
 
+    /**
+     * Provides access to the location of the installation
+     *
+     * @return location of the current installation or null if the location is not set
+     */
+    PointType getLocation();
```

Comment:
Normally I would prefer different services / providers for different stuff.

If we find a good name that combines Location and Locale I agree that this would be simpler to merge into one service instead of having two (as every service will provide only one function).

But I don't think we should merge `I18nProvider` and the `LocaleProvider` because the I18nProvider offers functions to localize text. The `LocaleProvider` itself holds a configuration.